src/commands/eightball.js

```diff
@@ -8,7 +8,11 @@ module.exports = {
    */
   command: function eightBall(message) {
     console.log('eightBall', message);
+    let options = ["As I see it, yes." , "Ask again later." , "Better not tell you now.", "Cannot predict now.", "Concentrate and ask again.", "Don't count on it.", "It is certain.", "It is decidedly so.", "Most likely.", "My reply is no.", "My sources say no.", "Outlook not so good.", "Outlook good.", "Reply hazy, try again.",
+"Signs point to yes.", "Very doubtful.", "Without a doubt.", "Yes.", "Yes - definitely.", "You may rely on it."]
+    let rand = Math.floor(Math.random() * 20)
```

Comment:
Change the `20` to `options.length`, this way if we add more options to the list we wont have to manually update the count.

Reply:
That's a good idea - I didn't anticipate that.